### PR TITLE
Deploy beam to NERSC

### PR DESCRIPTION
**Pull request by dimaopen**
One can use deployToNersc gradle task to deploy beam to nersc. The branch, commit, config, region, s3_publish is taken from gradle.deploy.properties. The AWS credentials and google API key is taken from the environment variables. One needs to use sshproxy.sh from NERSC to create the ssh key to login to NERSC.

---
This change is [Reviewable](https://reviewable.io/reviews/lbnl-ucb-sti/beam/3220)

### CODE REVIEW

**Comment by dimaopen** [comment]
Test!

**Review by JustinPihony — APPROVED** [review]
There should probably be something added to the docs about how to run this. Approving, as this is not a blocker, though

**Comment by dimaopen** [comment]
Please do not merge this PR, I need to add some more options in job scripts.

**Comment by JustinPihony** [comment]
I converted it to a draft so that it would not accidentally be merged.

**Comment by dimaopen** [comment]
Test!

**Comment by dimaopen** [comment]
Test!

**Comment by dimaopen** [comment]
Test!

File: docs/developers.rst [hunk 1]
```diff
@@ -195,6 +195,24 @@ Similarly for experiment batch, you can specify comma-separated experiment files
 
 For demo and presentation material, please follow the link_ on google drive.
 
+BEAM run on NERSC
+~~~~~~~~~~~~~~~~~
+
+In order to run BEAM on NERSC one needs to get an `ssh key <https://docs.nersc.gov/connect/mfa/#sshproxy>`_ that allows you to ssh to NERSC sstems without further authentication until the key expires (24 hours). You also need to specify your user name on NERSC in the following property: **nerscUser**, i.e::
```
Comment (JustinPihony) [inline docs/developers.rst:201]: typo sstems to systems?

**Comment by dimaopen** [comment]
Test!

**Pull request merged**